### ISSUE TITLE
Accessibility - Sigin Screen - Question about Images

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#5]
VoiceOver only reads the text “U. S. DOL SafetyNET App” and the text in the button and not anything for the DOL seal, the SafetyNET logo, of the Powered by CIO logo.

The images should have ALT text/description added if you wanted the user to know about these items and that it's powered by OCIO, etc.

**Comment by aanthonyDOL** [comment]
Added Voice Over for DOL seal, the SafetyNET logo, of the Powered by CIO logo

**Comment by jfc3-dol** [comment]
Please leave the issue open until I can check that the correction as been made.

**Comment by jfc3-dol** [comment]
Okay, that makes sense so I will close this issue.

**Comment by jfc3-dol** [comment]
Oops, typed in the wrong issue number so opening this one back up.
